### PR TITLE
replaced json file first commit

### PR DESCRIPTION
**Pull request by Jsm-mani**
please review the code @mupputur , @harisri9948 , @rajakumaro , @amlya1230 , @gopi023 , @anilkumar-golla , @chennakesava89

### CODE REVIEW

**Review by Rajkumar-V — APPROVED** [review]
code is good

File: json_parsers/json_parser4.py.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+import json
+
+    
+class Json:
```
Comment (amlya1230) [inline json_parsers/json_parser4.py.py:5]: Class name ideally should not be a built-in module names. And it should be followed camel case
Example
JsonReader

File: json_parsers/json_parser4.py.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        data=f.read()
+        y=json.loads(data)
+        f.close()
+        print(y)
+        print('#'*50)
+        z=y["markers"]
+        z[0]["location"]=z[0].pop("position")
+        return y
+    
+    def To_json(self,y):
```
Comment (amlya1230) [inline json_parsers/json_parser4.py.py:18]: 1. Usually the function names should start with lower case letters
2. The function should have return statement with proper exceptional handling if require

File: json_parsers/json_parser4.py.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+
+import json
+
+    
+class Json:
+
+    def json_convert(self,path):
+        f=open(input_path.strip("\u202a"),'r')
```
Comment (amlya1230) [inline json_parsers/json_parser4.py.py:8]: Where you have defined variable 'input_path'?

File: json_parsers/json_parser4.py.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        data=f.read()
+        y=json.loads(data)
+        f.close()
+        print(y)
```
Comment (amlya1230) [inline json_parsers/json_parser4.py.py:12]: define separate functions  for to get the markers and locations

File: json_parsers/json_parser4.py.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    def To_json(self,y):
+        self.x=json.dumps(y,indent=4)
+        print('#'*50)
+        print(self.x)
+
+
+
+if __name__=="__main__":
+    j=Json()
+    input_path="D:\\py\\dependencies\\json\\input1.json"
```
Comment (amlya1230) [inline json_parsers/json_parser4.py.py:27]: Please give the relative path from your project directory

File: json_parsers/json_parser4.py.py [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    out=j.json_convert(input_path)
+    print(out)
+    put=j.To_json(out)
+
+
```
Comment (amlya1230) [inline json_parsers/json_parser4.py.py:32]: Remove extra lines and commented code

**Review by amlya1230 — CHANGES_REQUESTED** [review]
Please find inline comments.

**Pull request closed without merging**